client/src/components/GameTimer.jsx

```diff
@@ -1,23 +1,27 @@
 import { useState, useEffect } from 'react';
 
-export default function GameTimer({ duration }) {
-    const [secondsElapsed, setSecondsElapsed] = useState(Math.floor(duration / 1000));
+export default function GameTimer({ time }) {
+    const [secondsElapsed, setSecondsElapsed] = useState(
+        Math.floor(time / 1000)
```

Comment:
Here you're taking a property (`time`) and creating a state from it, with the value of the property being the default (initial) value of the state.

The property itself is passed in from `App`, where it is stored in state as well. This means that you have two states representing the same actual state (the time of the game). That's probably not what you want.